src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -270,8 +270,8 @@ private void requestEntityToBuildingIfRequired(Template.EntityInfo entityInfo)
                     {
```

Comment:
also we should probably change that check to `stack != null && stack != ItemStack.EMPTY`, it looks like thats what it is initialized to: `this.getDataManager().register(ITEM, ItemStack.EMPTY);`